telemetry/roles/idrac_telemetry/tasks/idrac_telemetry_deployment.yml

```diff
@@ -100,10 +100,14 @@
                     value: mysqldb
                   - name: MYSQL_HOST_PORT
                     value: "{{ mysqldb_port1 }}"
+                  - name: HTTP_PROXY
+                    value: "{{ proxy[0].http_proxy | default('', true) }}"
+                  - name: HTTPS_PROXY
+                    value: "{{ proxy[0].https_proxy | default('', true) }}"
```

Comment:
NO_PROXY is missing here.

Reply:
@francares HTTP_PROXY and HTTPS_PROXY variable only required here and it provides internet.